packages/help-center/src/components/help-center-launchpad.tsx

```diff
@@ -43,6 +44,11 @@ export const HelpCenterLaunchpad = () => {
 		siteSlug = window?.location?.host;
 	}
 
+	const { data } = useLaunchpadChecklist( siteSlug, siteIntent || '' );
```

Comment:
Can we have the siteIntent's empty string fallback `|| ''` built into the hook itself. The drawback right now is we have to remember to pass a fallback for that value each time we reuse the hook